src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -147,7 +147,9 @@ export class TemplateActionService extends SyncServiceBase {
       log_groupEnd();
     }
     // resolve once full queue processed
-    await this.actionsQueueProcessing$.pipe(takeWhile((v) => v === true)).toPromise();
+    await lastValueFrom(this.actionsQueueProcessing$.pipe(takeWhile((v) => v === true)), {
+      defaultValue: "No action queue",
```

Comment:
nit(non-blocking)
Is there a reason for passing a default value?

Reply:
From my testing, if no value was provided for `defaultValue` then triggering an action would cause an `EmptyError` error to be thrown.

As outlined [here](https://rxjs.dev/deprecations/to-promise#lastvaluefrom), `lastValueFrom` differs slightly from `toPromise` in that when the observable completes without emitting a single value, the latter resolves successfully with value `undefined`, whereas the former rejects with `EmptyError`.

Without a default value, actions still appear to be triggered correctly, but the following error is thrown and logged to the console:
<img width="731" alt="Screenshot 2023-01-04 at 16 13 08" src="https://user-images.githubusercontent.com/64838927/210599858-cf5b9a36-f77b-4581-99d2-e98dc3339e55.png">